src/TraceEvent/Parsers/ClrTraceEventParser.cs

```diff
@@ -17,6 +18,31 @@ namespace Microsoft.Diagnostics.Tracing.Parsers
     using Microsoft.Diagnostics.Tracing.Parsers.Clr;
     using System.Collections.Generic;
 
+    // TODO, AndrewAu, naming
+    public class CommittedUsageTraceData
+    {
+        public long totalCommittedInUse;
```

Comment:
Probably best to add the units (KB/MB) to the names of each of these fields.

Reply:
Discerned that these are all in Bytes. @cshung, please correct me if this is not the case.